Makefile

```diff
@@ -42,12 +44,12 @@ test-reconciler:
 # Requires ko 0.2.0 or newer.
 test-images:
 	for img in $(TEST_IMAGES); do \
-		ko resolve --tags=latest -RBf $$img ; \
+		KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ko build --tags=$(TEST_IMAGE_TAG) $(KO_FLAGS) -B $$img ; \
 	done
 .PHONY: test-images
 
 test-image-single:
-	ko resolve --tags=latest -RBf test/test_images/$(IMAGE)
+	KO_DOCKER_REPO=$(DOCKER_REPO_OVERRIDE) ko build --tags=$(TEST_IMAGE_TAG) $(KO_FLAGS) -B test/test_images/$(IMAGE)
```

Comment:
When I try building some of the images using this command I'm getting this error:
```
make test-image-single DOCKER_REPO_OVERRIDE=quay.io/openshift-knative/eventing-kafka-broker/ KO_FLAGS="--platform=all --sbom=none" TEST_IMAGE_TAG=v1.4.0 IMAGE=kafka-consumer
KO_DOCKER_REPO=quay.io/openshift-knative/eventing-kafka-broker/ ko build --tags=v1.4.0 --platform=all --sbom=none -B test/test_images/kafka-consumer
Error: failed to publish images: importpath "ko://test/test_images/kafka-consumer" is not supported: importpath is not `package main`
2022/09/27 14:22:25 error during command execution:failed to publish images: importpath "ko://test/test_images/kafka-consumer" is not supported: importpath is not `package main`
```
What works for me is using `resolve` instead of `build` and using `-RBf` instead of just `-B` flag.